manifests/o/Opsfolio/Surveilr/0.9.0/Opsfolio.Surveilr.locale.en-US.yaml

```
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: Opsfolio.Surveilr
+PackageVersion: 0.9.0

```

```suggestion
PackageVersion: 0.9.1
```
Please update file path to match this change.